com.unity.render-pipelines.high-definition/Editor/PostProcessing/DepthOfFieldEditor.cs

```diff
@@ -80,68 +80,105 @@ public override void OnInspectorGUI()
             int mode = m_FocusMode.value.intValue;
             if (mode == (int)DepthOfFieldMode.Off)
                 return;
+            
+            using (new HDEditorUtils.IndentScope())
+            {
+                // Draw the focus mode controls
+                DrawFocusSettings(mode);
+            }
+
+            EditorGUILayout.Space();
 
             base.OnInspectorGUI();
 
-            bool advanced = isInAdvancedMode;
+            using (new HDEditorUtils.IndentScope())
+            {
+                // Draw the quality controls
+                GUI.enabled = GUI.enabled && base.overrideState;
+                DrawQualitySettings();
+            }
+
+            GUI.enabled = true;
+        }
 
+        void DrawFocusSettings(int mode)
+        {
             if (mode == (int)DepthOfFieldMode.UsePhysicalCamera)
             {
                 PropertyField(m_FocusDistance);
-
-                if (advanced)
-                {
-                    GUI.enabled = useCustomValue;
-                    EditorGUILayout.LabelField("Near Blur", EditorStyles.miniLabel);
-                    PropertyField(m_NearSampleCount, Styles.k_NearSampleCount);
-                    PropertyField(m_NearMaxBlur, Styles.k_NearMaxBlur);
-
-                    EditorGUILayout.LabelField("Far Blur", EditorStyles.miniLabel);
-                    PropertyField(m_FarSampleCount, Styles.k_FarSampleCount);
-                    PropertyField(m_FarMaxBlur, Styles.k_FarMaxBlur);
-                    GUI.enabled = true;
-                }
             }
             else if (mode == (int)DepthOfFieldMode.Manual)
             {
-                EditorGUILayout.Space();
+                EditorGUILayout.LabelField("Near Range", EditorStyles.miniLabel);
+                PropertyField(m_NearFocusStart, EditorGUIUtility.TrTextContent("Start"));
+                PropertyField(m_NearFocusEnd, EditorGUIUtility.TrTextContent("End"));
 
-                EditorGUILayout.LabelField("Near Blur", EditorStyles.miniLabel);
-                PropertyField(m_NearFocusStart, Styles.k_NearFocusStart);
-                PropertyField(m_NearFocusEnd, Styles.k_NearFocusEnd);
+                EditorGUILayout.LabelField("Far Range", EditorStyles.miniLabel);
+                PropertyField(m_FarFocusStart, EditorGUIUtility.TrTextContent("Start"));
```

Comment:
There was a bug where the tooltips for Far blur were wrong (https://fogbugz.unity3d.com/f/cases/1256641/), so I had to replace
PropertyField(m_FarFocusStart, EditorGUIUtility.TrTextContent("Start"));
with 
PropertyField(m_FarFocusStart, Styles.k_FarFocusStart); 

I see this change was reverted (probably during a merge), although you still have the Styles class definition.
Can you make sure that the tooltips for the Far Range are correct? If not, then you can get the tooltips/names from the Styles class.

Reply:
Ah yes, it likely happened when I merged the physically based DoF. Just pushed fix.